client/src/components/CreateRecipe.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import helperFunctions from '../helperFunctions';
+
+class CreateRecipe extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      // need to grab user id!!!
+      recipeName: '',
+      servingSize: 2,
+      cookingTime: 30,
+      ingredients: '',
+      direction: '',
+      images: [],
+      userId: 1,
+      previewImgs: [],
+      files: [],
+    };
+
+    this.updateForm = this.updateForm.bind(this);
+    this.submitForm = this.submitForm.bind(this);
+    this.previewImg = this.previewImg.bind(this);
+  }
+
+  updateForm (e) {
+    e.preventDefault();
+    const newState = this.state;
```

Comment:
Might want to create a copy rather than directly referencing the original object

Reply:
I am creating a copy of the state here.